docs/spark-procedures.md

```diff
@@ -493,6 +493,39 @@ CALL spark_catalog.system.add_files(
 )
 ```
 
+### `register_table`
+
+Creates a catalog entry for a metadata.json file which already exists but does not have a corresponding catalog identifier.
+
+#### Usage
+
+| Argument Name | Required? | Type | Description |
+|---------------|-----------|------|-------------|
+| `table`       | ✔️  | string | Table which is to be registered |
+| `metadata_file`| ✔️  | string | Metadata file which is to be registered as a new catalog identifier |
+
+{{< hint warning >}}
```

Comment:
I am not sure how this looks like, even in rich text view it didn't show as warning. 

But I know another way to give warning.
Adding as "!!! note" will display as a block in the sitedocs. 
Example:
md file: https://github.com/projectnessie/nessie/blob/main/site/docs/tools/iceberg/spark.md#spark-via-iceberg
respective site doc: https://projectnessie.org/tools/iceberg/spark/#spark-via-iceberg

So, According to me the below format is good. Let's see what @RussellSpitzer thinks.

!!! note
Having the same metadata.json registered in more than one catalog can lead to missing updates, loss of data, and table corruption. Only use this procedure when the table is no longer registered in an existing catalog, or you are moving a table between catalogs.

Reply:
Hint warning is used in the Iceberg hive docs: https://github.com/apache/iceberg/blob/master/docs/hive.md. It displays a yellow block: https://iceberg.apache.org/docs/latest/hive/

I have kept `!!! note` for time being.

Reply:
Sorry for the little back and forth. If `Hint warning` is already used in our docs. Then it should be fine to use and the document style also will be uniform.

Reply:
No worries, I have updated it back to Hint warning.

Reply:
@RussellSpitzer: we discussed Hint warning syntax here.